core/lib/customer-login-api.ts

```diff
@@ -0,0 +1,57 @@
+import { randomUUID } from 'crypto';
+import { SignJWT } from 'jose';
+
+/**
+ * Build a Customer Login API JWT which can be used in auth.ts to log in a customer
+ * using the LoginWithTokenMutation, or used as a redirect to /login/token/[token]
+ *
+ * This is a stub intended to be used when implementing 3rd party authentication callbacks
+ *
+ * Requires that BIGCOMMERCE_CLIENT_SECRET and BIGCOMMERCE_CLIENT_ID are set in the environment
+ * from a client that has the Customer Login scope enabled
+ *
+ * @param {number} customerId - The BigCommerce customer ID to generate the login token for
+ * @param {number} [channelId] - Optional channel ID that the customer will be logged into
+ * @param {string} [redirectTo] - Optional relative URL to redirect to after successful login
+ * @returns {Promise<string>} A JWT token that can be used to authenticate the customer
+ * @throws {Error} If BIGCOMMERCE_CLIENT_SECRET is not set in environment variables
+ * @throws {Error} If BIGCOMMERCE_CLIENT_ID is not set in environment variables
+ */
+const generateCustomerLoginApiJwt = async (
```

Comment:
Can we remove the default export in favor of the named export?
```suggestion
export const generateCustomerLoginApiJwt = async (
```